changelog_unreleased/blog-post-intro.md

```diff
@@ -0,0 +1,15 @@
+---
+author: "Sosuke Suzuki (@sosukesuzuki)"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 2.8: improve --cache CLI option and TypeScript 4.9 syntax!"
```

Comment:
```suggestion
title: "Prettier 2.8: improve --cache CLI option and TypeScript 4.9 satisfies operator!"
```

```suggestion
title: "Prettier 2.8: improve `--cache` CLI option and TypeScript 4.9 `satisfies` operator!"
```

Might be not worth the change if we plan to release 2.8.1 with Auto-Accessors in classes. On the other hand, this will align blog title with the section title and avoid overpromises.